components/Breadcrumbs.test.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import Breadcrumbs from './Breadcrumbs.tsx'
+import '@testing-library/jest-dom'
```

Comment:
Imported to be able to use `expect(...).toBeInTheDocument() `

Reply:
Add this as a comment in the file.